pages/linux/btrfs-device.md

```diff
@@ -0,0 +1,24 @@
+# btrfs device
+
+> Manage devices in a btrfs filesystem.
+> More information: <https://btrfs.wiki.kernel.org/index.php/Manpage/btrfs-device>.
+
+- Add one or more devices to a btrfs filesystem:
+
+`sudo btrfs device add {{path/to/block_device}} [{{...}}] {{path/to/btrfs_filesystem}}`
```

Comment:
@sbrl is this official tldr-pages syntax for an arbitrary number of parameters?
What about something like this:
```suggestion
`sudo btrfs device add {{path/to/block_device1}} {{path/to/block_device2}} {{path/to/btrfs_filesystem}}`
```
?

Reply:
I think I've seen it done somewhere before, but I can't remember.

Reply:
I like my proposal.  Can we do it?

Reply:
I'd suggest enclosing the 2nd block device in square brackets to indicate it's optional. We have done that elsewhere.

Reply:
I've updated my comment.